plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/BaseHudiConnectorTest.java

```diff
@@ -93,9 +79,7 @@ public void testShowCreateTable()
                         ")");
     }
 
-    protected abstract HoodieTableType getHoodieTableType();
-
-    static String columnsToHide()
+    protected static String columnsToHide()
```

Comment:
Why does this need to be explicitly set? If we always want to hide these columns we should either change default value of the config or stop hiding these in tests - it's easy for some bug to get missed this way.

Reply:
Agree that hiding columns in default config or base test class may hide some issues, but I feel we tend to override and ignore tests with TODO comment when adding tests to base class that requires some customization in the connector side. Adding dedicated tests for meta columns in #14427 and keep hiding them looks good to me.